### PR TITLE
Active loading

### PR DESCRIPTION
**Pull request by nolds9**
Used `bullet` gem to find extra `n+1` queries, fixed by utilizing eager-loading

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -38,4 +38,27 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+
+  config.after_initialize do
+    Bullet.enable = true
+    Bullet.alert = true
```
Comment (nolds9) [inline config/environments/development.rb]: Can deactivate in development if alert becomes annoying

**Comment by jshawl** [comment]
what does this mean?!

**Comment by jshawl** [comment]
Is there a test we can write to show this optimization? if not that's cool 👍

**Comment by nolds9** [comment]
Doubt there's an easy way to test it considering its just measuring queries. Here's a little demo if interested: https://github.com/flyerhzm/bullet#demo

**Comment by nolds9** [comment]
@adambray

**Pull request merged**